pkg/apis/hub/v1alpha1/api.go

```diff
@@ -29,17 +28,12 @@ import (
 // +kubebuilder:printcolumn:name="PathPrefix",type=string,JSONPath=`.spec.pathPrefix`
```

Comment:
The additional printer columns `PathPrefix`, `ServiceName` and `ServicePort` are no longer needed.